spyder_vim/vim_widget.py

```diff
@@ -250,12 +254,15 @@ class VimLineEdit(QLineEdit):
     def keyPressEvent(self, event):
         if event.key() == Qt.Key_Escape:
             self.clear()
+        elif event.key() == Qt.Key_Backspace:
+            self.setText(self.text() + "\b")
         else:
             QLineEdit.keyPressEvent(self, event)
 
     def focusInEvent(self, event):
         QLineEdit.focusInEvent(self, event)
         self.parent().update_vim_cursor()
+        self.parent().vim_keys("h", 1)
```

Comment:
You're right, sorry. In that case the best would be to call directly `self.parent().h()` without changing `__call__` at all.